src/pvrclient-argustv.cpp

```diff
@@ -709,8 +709,13 @@ PVR_ERROR cPVRClientArgusTV::GetRecordings(bool deleted,
             {
               kodi::addon::PVRRecording tag;
 
-              tag.SetSeriesNumber(recording.SeriesNumber());
-              tag.SetEpisodeNumber(recording.EpisodeNumber());
+              //There may be cases where series and/or episode are populated withe 0 by default
+              //if neither value is more than 0, there is no value to use or show them
+              if(recording.SeriesNumber() > 0 || recording.EpisodeNumber() >0)
```

Comment:
Coding guidelines: blank after ‚if‘ and blank after ,>‘, please

Reply:
> Coding guidelines: blank after ‚if‘ and blank after ,>‘, please

Sorry, I had cleaned that up in the Matrix PR, but failed to clean it up here.  It is cleaned up now.